references/cli/system.go

```diff
@@ -0,0 +1,175 @@
+		Long:  "Manage system, incluing printing the system deployment information in vela-system namespace and diagnosing the system's health.",
+		Example: "# Check all deployments information:\n" +
+			"> vela system info\n" +
+			"# Specify a deployment name to check detail information:\n" +
+			"> vela system info --name kubevela-vela-core\n" +
+			"# Diagnose the system's health:\n" +
+			"> vela system diagnose\n",
+		Annotations: map[string]string{
+			types.TagCommandType: types.TypeSystem,
+		},
+	}
+	cmd.AddCommand(
+		NewSystemInfoCommand(c),
+		NewSystemDiagnoseCommand(c))
+	return cmd
+}
+
+// NewSystemInfoCommand prints system detail info
+func NewSystemInfoCommand(c common.Args) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "info",
+		Short: "Print the system deployment detail information in vela-system namespace.",
+		Long:  "Print the system deployment detail information in vela-system namespace.",
+		Args:  cobra.ExactValidArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Get deploymentName from flag
+			deployName, err := cmd.Flags().GetString(FlagDeploymentName)
+			if err != nil {
+				return errors.Wrapf(err, "failed to get deployment name flag")
+			}
+			table := newUITable().AddRow("NAME", "NAMESPACE", "IMAGE", "ARGS")
+			table.MaxColWidth = 120
+			config, err := c.GetConfig()
+			if err != nil {
+				return err
+			}
+			// Get clientset
+			clientset, err := kubernetes.NewForConfig(config)
+			if err != nil {
+				panic(err)
+			}
+			// Get deploymentsClient
+			deploymentsClient := clientset.AppsV1().Deployments(types.DefaultKubeVelaNS)
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			if deployName != "" {
+				// DeployName is not empty, print the specified deployment's all args
+				deployment, err := deploymentsClient.Get(ctx, deployName, metav1.GetOptions{})
+				if err != nil {
+					panic(err)
+				}
+				table.AddRow(deployment.Name, deployment.Namespace, deployment.Spec.Template.Spec.Containers[0].Image, strings.Join(deployment.Spec.Template.Spec.Containers[0].Args, " "))
+			} else {
+				// DeployName is empty, print all deployment's partial args
+				deployments, err := deploymentsClient.List(ctx, metav1.ListOptions{})
+				if err != nil {
+					panic(err)
+				}
+				for _, deploy := range deployments.Items {
+					allArgs := deploy.Spec.Template.Spec.Containers[0].Args
+					table.AddRow(deploy.Name, deploy.Namespace, deploy.Spec.Template.Spec.Containers[0].Image, limitStringLength(strings.Join(allArgs, " "), 60))
+				}
+			}
+			if len(table.Rows) == 1 {
+				cmd.Println("No deployment found.")
+			} else {
+				cmd.Println(table.String())
+			}
+			return nil
+		},
+		Annotations: map[string]string{
+			types.TagCommandType: types.TypeSystem,
+		},
+	}
+	cmd.Flags().StringP(FlagDeploymentName, "n", "", "Specify the deployment name to check detail information. If empty, it will print all deployments information. Default to be empty.")
```

Comment:
We typically think `-n` as `namespace`. Consider choosing a different letter or use non-flag arguments.